From ebd59d271c24601e08c3569681b129cd27bf4070 Mon Sep 17 00:00:00 2001
From: Michael Niedermayer <michaelni@gmx.at>
Date: Sat, 7 Mar 2015 09:59:27 +0100
Subject: [PATCH] Revert "avutil/opencl: is_compiled flag not being cleared in
 av_opencl_uninit"

Fixed build with opencl enabled
Found-by:  WJ Liu

This reverts commit 0f2359b86926ed33da4bd64ca76d84d03d5ad380.
---
 libavutil/opencl.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/libavutil/opencl.c b/libavutil/opencl.c
index a3866ca..36cb6fe 100644
--- a/libavutil/opencl.c
+++ b/libavutil/opencl.c
@@ -611,9 +611,6 @@ void av_opencl_uninit(void)
         }
         opencl_ctx.context = NULL;
     }
-    for (i = 0; i < opencl_ctx.kernel_code_count; i++) {
-        opencl_ctx.kernel_code[i].is_compiled = 0;
-    }
     free_device_list(&opencl_ctx.device_list);
 end:
     if (opencl_ctx.init_count <= 0)
-- 
2.3.2

